routing/src/prefix.rs

```diff
@@ -121,10 +131,68 @@ impl Display for Prefix {
     }
 }
 
+impl Serialize for Prefix {
```

Comment:
Why do you need this for? It's absolutely fine, since I was planning to add something similar.... I'm just curious.

Reply:
Right now, only for tests. We have a test in the nat module that parses the VPCs and PIFs from a YAML file - from the early experiments in the tmp-nat repo. In order to parse the file, I need the objects to implement `Deserialize`. Technically, we don't use `Serialize` at the moment, but having just `Deserialize` and not `Serialize` for the object felt a bit silly.

I'm not particularly proud of the serializer I've come up with, if you've got something more robust feel free to improve or replace, by all means!

Reply:
It's fine, I was just curious. Thanks!